frontend/packages/console-dynamic-plugin-sdk/src/app/__tests__/AppInitSDK.spec.tsx

```diff
@@ -1,113 +1,90 @@
-import { shallow, mount } from 'enzyme';
-import { Provider } from 'react-redux';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import * as reactRedux from 'react-redux';
 import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 import AppInitSDK from '../AppInitSDK';
 import * as configSetup from '../configSetup';
 import * as apiDiscovery from '../k8s/api-discovery/api-discovery';
 import * as hooks from '../useReduxStore';
 
-jest.mock('react-redux', () => {
-  const ActualReactRedux = jest.requireActual('react-redux');
-  return {
-    ...ActualReactRedux,
-    useStore: jest.fn(),
-  };
-});
+jest.mock('react-redux', () => ({
+  Provider: jest.fn(({ children }) => children),
+}));
+
+jest.mock('../useReduxStore', () => ({
+  useReduxStore: jest.fn(),
+}));
+
+jest.mock('../configSetup', () => ({
+  setUtilsConfig: jest.fn(),
+}));
+
+jest.mock('../k8s/api-discovery/api-discovery', () => ({
+  initApiDiscovery: jest.fn(),
+}));
+
+const { useReduxStore: useReduxStoreMock } = hooks as jest.Mocked<typeof hooks>;
+const mockStore = configureMockStore([thunk]);
+const store = mockStore({});
+const mockProvider = (reactRedux as jest.Mocked<typeof reactRedux>).Provider;
+const ChildComponent = () => <div>Hello, OpenShift!</div>;
+const mockConfig = { appFetch: jest.fn() };
+const mockApiDiscoveryConfig = { apiDiscovery: jest.fn(), appFetch: jest.fn() };
 
 describe('AppInitSDK', () => {
-  const mockStore = configureMockStore();
-  const store = mockStore([thunk]);
-  const mockConfig = {
-    apiDiscovery: jest.fn(),
-    appFetch: jest.fn(),
+  const renderAppInitSDK = (config, storeContext = { store, storeContextPresent: true }) => {
+    useReduxStoreMock.mockReturnValue(storeContext);
+    return render(
+      <AppInitSDK configurations={config}>
+        <ChildComponent />
```

Comment:
ChildComponent feels redundant here if its not changed at all

```suggestion
        <div>Hello, OpenShift!</div>
```